lib/index.js

```diff
@@ -0,0 +1,11 @@
+
+const Util = require('util');
+
+const Config = require('../config');
+const MoviesServer = require('./server');
+
+MoviesServer.start(() => {
+	Util.log(`MoviesServer started on port:${Config.PORT}`);
+});
+
+
```

Comment:
While we like having the new line at the end of the file, we like to make sure that there aren't any "stray" new lines throughout the codebase.